### PR TITLE
Nitpicks

### PR DESCRIPTION
**Pull request by adi1133**
Some minor cleanups.

### CODE REVIEW

File: demo/src/main/java/com/bluelinelabs/conductor/demo/MainActivity.java [hunk 1]
```diff
@@ -12,22 +12,20 @@
...
     private Router router;
-    private Unbinder unbinder;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
         setContentView(R.layout.activity_main);
-        unbinder = ButterKnife.bind(this);
+        ButterKnife.bind(this);
```
Comment (EricKuck) [inline demo/src/main/java/com/bluelinelabs/conductor/demo/MainActivity.java]: Why remove the unbinder?

Comment (adi1133) [inline demo/src/main/java/com/bluelinelabs/conductor/demo/MainActivity.java]: All references to the activity (and its views) are released after onDestory(). The unbinder is not required.

Comment (dimsuz) [inline demo/src/main/java/com/bluelinelabs/conductor/demo/MainActivity.java]: Imho, it doesn't hurt to release them manually, also shows off this ButterKnife feature (which is a bit new if I'm right)
Also helps to get used to using it in Controllers... Maybe a comment about not-being-required here would help. 

On the other hand it is also shown off in ButterKnifeController... :)

Comment (adi1133) [inline demo/src/main/java/com/bluelinelabs/conductor/demo/MainActivity.java]: In ButterKnifeController it is required to not leak memory.

**Pull request merged**